services/src/main/java/io/druid/cli/CliCoordinator.java

```diff
@@ -255,6 +255,6 @@ public LoadQueueTaskMaster getLoadQueueTaskMaster(
 
   public static boolean isOverlord(Properties properties)
   {
-    return Boolean.valueOf(properties.getProperty("druid.coordinator.asOverlord.enabled")).booleanValue();
+    return Boolean.valueOf(properties.getProperty("druid.coordinator.asOverlord.enabled"));
```

Comment:
Boolean.parseBoolean returns a primitive

Reply:
There is no need for the `.booleanValue()` method, cuz Java will automatically unbox `Boolean` primitive after JDK 5+.

Reply:
I mean there is no need to get a boxed Boolean either, `Boolean.parseBoolean()` is a direct equivalent which returns a primitive.

Reply:
I see. I will fix it.

Reply:
Done.